apps/zaberCtrl/zaberCtrl.hpp

```diff
@@ -192,8 +192,9 @@ void zaberCtrl::loadConfig()
 
 int zaberCtrl::appStartup()
 {
-   
-   createStandardIndiNumber<float>( m_indiP_pos, "position", 0.0, std::numeric_limits<float>::max(), 0.0, "%0.3d");
+
+   // units of mm, stage has ~ 1/10 um of precision but 0.1 mm is a sensible small step size
```

Comment:
I say we just use 1/m_countsPerMillimeter as the step size.
1) It's stage dependent, so this ought to be based on "m_countsPerMillimeter".  The `stagscibs` and `stageeyepiece` are half the stepsize.
2) There's an inconsistency with setting this to 0.1, because the stage can't actually move 0.1.  It can only move 1/m_countsPerMillimeter steps.

Reply:
Hmm, I agree from a theoretical purity standpoint, but I think the only thing using the step size currently is the web UI and having step buttons that step by 0.00009 mm isn't very useful.

Reply:
well neither is 0.0001.  if this only used for the type of stepsizes that a human is interested in, it's probably 0.1 or 1.0. but how do you communicate to a script, which might care at this level (if it's doing precision alignment, say)?

Reply:
The step size in the following line is 0.1, not 0.0001. The formatting precision is to 0.0001. We could add more properties to the stages that report their human-friendly step sizes, but that seems out of scope for this change.

Reply:
Ah, right, 0.1 mm, I see what you're trying to do.  But, I don't interpret the `step` as a GUI hint.  It's the "allowed increment", so isn't this a statement by this stage that "I will only move in increments of this size"?  It's the format that should be `%0.1f` and interpreted by the GUI as a hint to what a human is interested in, right?

Reply:
No, the format string is the hint to the client library for how to parse values off the wire / out of the XML text body (e.g. with `sscanf`). That's why I wanted to modify this in the first place: it was only moving in integer steps when parsing / serializing according to the `format=` attribute, which was specifying `%0.3d` (an integer format, though an unusual one).

If you're doing precision alignment, and you want to move in steps that are guaranteed to be one step at the stage, you'd be using rawpos anyway, right?

We can also leave the step size at 0.0, like it was, but I think you'd agree that's _also_ wrong :) I don't think this PR is the absolute last word on zaber stage properties, but I think it'll fix two issues we saw yesterday with the current controller.